src/dunst.c

```diff
@@ -229,10 +255,16 @@ int dunst_main(int argc, char *argv[])
         log_set_level_from_string(verbosity);
         g_free(verbosity);
 
-        char *cmdline_config_path;
-        cmdline_config_path =
-            cmdline_get_string("-conf/-config", NULL,
-                               "Path to configuration file");
+        cmdline_usage_append("-conf/-config", "string", "Path to configuration file");
+
+        int start = 1, count = 1;
+        while (cmdline_get_string_offset("-conf/-config", NULL, start, &start))
+                count++;
+
+        config_paths = g_malloc0(sizeof(char *) * count);
```

Comment:
Is this freed properly somewhere?

Reply:
In the teardown function. And when it is overridden in reload